Repository Interface Cleanup

vjkoskela opened the pull request:
Rename repository interface methods to be more consistent with each other and avoid ambiguity/collisions (e.g. get on report and job).

BrandonArp commented on app/com/arpnetworking/kairos/client/KairosDbClient.java:
```diff
@@ -31,9 +31,9 @@
 public interface KairosDbClient {
     /**
-     * Executes a query for datapoints from  KairosDB.
+     * Executes a queryAlerts for datapoints from  KairosDB.
```
I don't think changes in this file were intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/KairosDbClientImpl.java:
```diff
@@ -178,9 +178,9 @@ private KairosDbClientImpl(final Builder builder) {
     private final FiniteDuration _readTimeout;
 
-    static final URI METRICS_QUERY_PATH = URI.create("/api/v1/datapoints/query");
+    static final URI METRICS_QUERY_PATH = URI.create("/api/v1/datapoints/queryAlerts");
```
I don't think changes in this file were intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/Aggregator.java:
```diff
@@ -28,7 +28,7 @@
 import javax.annotation.Nullable;
 
 /**
- * Model class to represent the aggregator in a metrics query.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/Metric.java:
```diff
@@ -29,7 +29,7 @@
 
 /**
- * Holds the data for a Metric element of the query.
+ * Holds the data for a Metric element of the queryAlerts.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/MetricsQuery.java:
```diff
@@ -34,13 +34,13 @@
 
 /**
- * Model class to represent a metrics query.
+ * Model class to represent a metrics queryAlerts.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/MetricsQueryResponse.java:
```diff
@@ -33,7 +33,7 @@
 
 /**
- * Model class to represent a metrics query response from KairosDB.
+ * Model class to represent a metrics queryAlerts response from KairosDB.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/Rollup.java:
```diff
@@ -21,7 +21,7 @@
 
 /**
- * Defines a query and destination for a RollupTask.
+ * Defines a queryAlerts and destination for a RollupTask.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/kairos/client/models/RollupQuery.java:
```diff
@@ -57,7 +57,7 @@ public Builder() {
 
         /**
-         * Sets the relative start time of the query. Required. Cannot be null.
+         * Sets the relative start time of the queryAlerts. Required. Cannot be null.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/AlertRepository.java:
```diff
@@ -56,23 +56,23 @@
 
     /**
-     * Create a query against the alerts repository.
+     * Create a queryAlerts against the alerts repository.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java:
```diff
@@ -55,7 +55,7 @@
      * Public constructor.
      *
-     * @param cassandraSession a Session to use to query data
+     * @param cassandraSession a Session to use to queryAlerts data
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java:
```diff
@@ -116,17 +116,17 @@ public int delete(final UUID identifier, final Organization organization) {
         assertIsOpen();
         LOGGER.debug()
-                .setMessage("Preparing query")
+                .setMessage("Preparing queryAlerts")
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -124,24 +124,24 @@ public void close() {
         assertIsOpen();
         LOGGER.debug()
-                .setMessage("Preparing query")
+                .setMessage("Preparing queryAlerts")
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -124,24 +124,24 @@ public void close() {
         LOGGER.debug()
                 .setMessage("Querying")
-                .addData("query", query)
+                .addData("queryAlerts", query)
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -124,24 +124,24 @@ public void close() {
                 .log();
 
-        // Create the base query
+        // Create the base queryAlerts
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -296,7 +296,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
 
     /**
-     * Inteface for database query generation.
+     * Inteface for database queryAlerts generation.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -305,7 +305,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
          * @param repository The alert repository instance.
          * @param query The repository agnostic <code>AlertQuery</code>.
-         * @return The database specific <code>PagedList</code> query result.
+         * @return The database specific <code>PagedList</code> queryAlerts result.
```
I don't think this change was intended.

BrandonArp commented on app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java:
```diff
@@ -328,7 +328,7 @@ private NagiosExtension convertToEbeanNagiosExtension(@Nullable final models.int
 
     /**
-     * RDBMS agnostic query for alerts.
+     * RDBMS agnostic queryAlerts for alerts.
```
I don't think this change was intended.

vjkoskela commented:
Apologies for that, looks like my IDE got a little carried away. I've replaced it with a version where I just replaced the method calls (setting on the refactoring UI) and checked the incremental diff of each method rename to ensure there was no funny business.
